src/lib/bootloader/none_bootloader.rb

```diff
@@ -0,0 +1,19 @@
+require "yast"
+
+Yast.import "HTML"
+
+module Bootloader
+  # Represents when bootloader want user manage itself
+  class NoneBootloader < BootloaderBase
+    def summary
```

Comment:
This should be declared in BootloaderBase together with a note about the non-obvious fact  that the return type is an array.